ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml

```diff
@@ -566,6 +565,15 @@ presubmits:
         - --target=e2e-gcp-ovn-upgrade
```

Comment:
One more thing: to avoid conflicts (will happen only in rehearsals, but that's still annoying), add:

```
--input-hash=$(JOB_NAME)
```

to the ci-operator args in each of the presubmit jobs (NOT the generated periodics)

Reply:
done